backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHIteratorApi.scala

```diff
@@ -230,11 +230,12 @@ class CHIteratorApi extends IteratorApi with Logging with LogLevelUtil {
   /**
    * Generate closeable ColumnBatch iterator.
    *
+   * FIXME: This no longer overrides parent's method
```

Comment:
keep it in the `CHIteratorApi`

Reply:
Any suggested change? I only removed `override` modifier. But not sure where to put this method yet.

Reply:
> Any suggested change? I only removed `override` modifier. But not sure where to put this method yet.

This change is OK to me.